### PR TITLE
[Cppcheck] Support "Bug hunting"

### PR DESCRIPTION
**Pull request by suetsuguk**
Fix https://github.com/sider/runners/issues/1138

### CODE REVIEW

File: docs/tools/cplusplus/cppcheck.md [hunk 1]
```diff
@@ -53,6 +54,7 @@ You can use the following options to fine-tune Cppcheck to your project.
 | [`project`](#project)                                                                 | `string`             | -       |
 | [`language`](#language)                                                               | `string`             | -       |
 | [`addon`](#addon)                                                                     | `string`, `string[]` | -       |
+| [`bug-hunting`](#bug-hunting)                                                         | `boolean`            | -       |
```
Comment (ybiquitous) [inline docs/tools/cplusplus/cppcheck.md:57]: [IMO] To make the default value `false` may be understandable for users... 🤔

Comment (suetsuguk) [reply docs/tools/cplusplus/cppcheck.md]: Good catch!
I set the default value to false.
commit 78dba3438c935171e0b200dabd4983fb311e1ae2

As we discussed, we will change the default value to `true` after testing some projects.

**Comment by ybiquitous** [comment]
Oh, this feature is not released yet... I'll revert it. 😓

**Comment by suetsuguk** [comment]
I am sorry I caused you trouble.

**Comment by ybiquitous** [comment]
Never mind! 😄

**Pull request merged**